### DISCUSSION TITLE
Commit del Suggerimento 02EPM

### DISCUSSION THREAD

**Discussion by johnnyvisualstudio** [sidelux/LootBot#4] [General]
Non posso fare l'update dei file, quindi ne ho creato uno nuovo con lo stesso nome di "LootBot", per implementare il suggerimento 02EPM che era stato implementato solo in parte

**Comment by sidelux** [comment]
Vedo il fork che hai creato ma il commit è al punto in cui si trova la versione attuale del bot, probabilmente devi ancora fare push delle tue modifiche?
